### DISCUSSION TITLE
ummmm

### DISCUSSION THREAD

**Discussion by elronixxdxd** [storaged-project/udisks#1229] [Q&A] [answered]
Hello, I'm a new user, I think I have a problem, I don't remember installing any program, I'm familiar with this, and a connected device appeared trying to log in as superuser (I use Huayra/Debian).
Is this something bad or am I just made up?

**Comment by vojtechtrefny** [comment, marked as answer]
UDisks is usually part of default installation in most distributions because all major desktop environments use it for device management. UDisks itself is a daemon that already runs with root privileges. When you connect a device and try to mount it (in some cases this might happen automatically, it depends on your system configuration) UDisks will ask for your passphrase through [polkit](https://wiki.archlinux.org/title/Polkit) for authentication -- mounting a filesystem is a privileged operation which is in some special cases allowed without authentication (for example with removable device with a known filesystem) for convenience, but it general it isn't allowed so UDisks ask for authentication.
